astropy/coordinates/builtin_frames/intermediate_rotation_transforms.py

```diff
@@ -22,17 +22,18 @@
 
 # # first define helper functions
 def gcrs_to_cirs_mat(time):
-    #felestial-to-intermediate matrix
-    return erfa.c2i06a(time.jd1, time.jd2)
+    #celestial-to-intermediate matrix
+    return erfa.c2i06a(time.tt.jd1, time.tt.jd2)
 
 def cirs_to_itrs_mat(time):
     #compute the polar motion p-matrix
     xp, yp = get_polar_motion(time)
-    sp = erfa.sp00(time.jd1, time.jd2)
+    sp = erfa.sp00(time.tt.jd1, time.tt.jd2)
     pmmat = erfa.pom00(xp, yp, sp)
 
     #now determine the Earth Rotation Angle for the input obstime
-    era = erfa.era00(time.jd1, time.jd2)
+    # this should be UT1 used here.
+    era = erfa.era00(time.ut1.jd1, time.ut1.jd2)
```

Comment:
Same as above, do `time = time.ut1`.

But also: this will raise an exception outside of the IERS table range. Is that OK? If not, one has to go via `utils.get_dut1utc`, i.e., something like:
```
if time.scale != 'ut1':
    time.delta_ut1_utc = utils.get_dut1utc(time)
    time = time.ut1
```
cc @eteq, since he probably has more detail.

Also, here of course it is silly to re-use `time`; better to just call it `time_ut1` and `time_tt`.